### ISSUE TITLE
Another bug when following run_example directions

### ISSUE THREAD

**Issue by martinholmer** [PSLmodels/OG-Core#781]
When I try to work around the bug reported in issue #780, I get the following differences from the expected output of the example:

```
(ogcore-dev) run_examples% ls -l *csv
-rw-r--r--@ 1 mrh  staff  1584 Jan 21 15:34 expected_ogusa_example_output.csv
-rw-r--r--  1 mrh  staff  1640 Feb  6 19:56 ogcore_example_output.csv

(ogcore-dev) run_examples% diff ogcore_example_output.csv expected_ogusa_example_output.csv 
1,7c1,7
< ,Variable,2021,2022,2023,2024,2025,2026,2027,2028,2029,2030,2021-2030,SS
< 0,GDP ($Y_t$),-0.16735054602134064,-0.14157894238355173,-0.21744030762126618,-0.2228052844188786,-0.21053121804236263,-0.18645099156078335,-0.16038127595033963,-0.1227400770701889,-0.0966153460839421,-0.0686338595948621,-0.16109691147992355,-0.38614414003818304
< 1,Consumption ($C_t$),-0.0758758816984031,1.0241899555137248,0.12310086751513638,-0.12812369754705397,-0.24351352511095206,-0.32277894213513253,-0.3650822883087033,-0.4194900817886846,-0.4286449823367036,-0.43497670601333205,-0.12405815155773192,-0.7176647471705253
< 2,Capital Stock ($K_t$),-0.37345099560834866,-0.4088108703473829,-0.6526898850168946,-0.7238091470195364,-0.7435953157488909,-0.7329467510562223,-0.7061826074090103,-0.6395066073284759,-0.5915818248341471,-0.5359026282768895,-0.6114964655711641,-1.5067823096305364
< 3,Labor ($L_t$),-0.05611097430623594,0.0026962912189517642,0.0178107834416318,0.048116426639532126,0.07779800378985717,0.10917626595672396,0.13487374038967076,0.1567571028593383,0.17105135740113175,0.18400991222994992,0.08132170577305775,0.22254945389625472
< 4,Real interest rate ($r_t$),-3.0520263383185156,-2.9453224967669973,-2.6454456597459677,-2.528842883299367,-2.472932542916348,-2.458652343235198,-2.460178345199604,-2.587338980604573,-2.616194455888617,-2.658686368570325,-2.645123883468417,-1.4021961225701403
< 5,Wage rate,-0.11126945384732671,-0.14423721606614195,-0.23516455599464775,-0.2707397075510938,-0.2880473874416187,-0.2952417708139666,-0.29478870611867075,-0.2789852586535098,-0.26713231749569566,-0.25209842003726,-0.24416757465891017,-0.607341307944457
---
> ,2020,2021,2022,2023,2024,2025,2026,2027,2028,2029,2020-2029,SS
> GDP ($Y_t$),-0.9832819726616429,-0.9933501251124207,-1.0012567410309168,-0.8619138689893194,-0.8688491315615962,-0.7954144519971328,0.16253581034984757,0.2192235803044291,0.34635236458521806,0.4438971020754922,-0.004407434795898249,-0.28714100248737645
> Consumption ($C_t$),-0.19127695030852182,-0.43858394506951565,-0.6288263816283477,-0.775297529368063,-0.8956242449906628,-0.9756499196546723,-0.854088421002668,-0.760260341447571,-0.6828219249152719,-0.622154622256172,-0.00683407489394566,-0.4132474874303411
> Capital Stock ($K_t$),0.0,-0.27133059955486166,-0.4644215120292453,-0.5989068345527114,-0.6429241686471642,-0.657352677351997,-0.6282759118350663,-0.327506178627711,-0.0520530895270575,0.22674410523649707,-0.0034423952769236177,-1.199948270903317
> Labor ($L_t$),-1.5086442720954116,-1.3799019355333633,-1.2890706763826723,-1.0032119937562165,-0.9902783163382312,-0.8696900923213652,0.5909222126362949,0.5147888056287747,0.5614363434496884,0.560889081941133,-0.004936241462388058,0.20785660530047687
> Real interest rate ($r_t$),-4.946074273142971,-4.518628301408074,-4.21475567456401,-3.7582757475400683,-3.69628288281014,-3.548635621551941,-1.9754843403167293,-2.391993084674715,-2.6409620482126543,-2.9427138949486453,-0.034709775717813314,-1.790330045029657
> Wage rate,0.53346692125439,0.3920004196645291,0.29160614327237255,0.14275150363838113,0.12265017901625362,0.07491797368546342,-0.4258948297161163,-0.29409587149628164,-0.2139472878444211,-0.11642425896171027,0.0004993836471175186,-0.4939722514914293

(ogcore-dev) run_examples% 
```

The directions led me to think that I should expect no differences.  Is that correct?
If so, why do you think I'm getting massive differences?

The example ran smoothly using all four cores on my iMac in about 80 minutes.   There was no indication that anything was amiss.

**Comment by martinholmer** [comment]
Recently in PR #775, @rickecon said (emphasis added):

> * <b>test_run_example.py</b>  The [call_run_ogcore_example()](https://github.com/PSLmodels/OG-Core/blob/master/ogcore/tests/test_run_example.py#L14) function in lines 14-24 loads the run_ogcore_example.py file into the environment as a module. Then the [test_run_ogcore_example()](https://github.com/PSLmodels/OG-Core/blob/master/ogcore/tests/test_run_example.py#L28) function uses the multiprocessing.Process() function to call the module and run it. <b>The test just makes sure that the script (module) runs for 5 minutes without an error.</b>

@jdebacker,  This seems to be the reason for the problems reported in issues #780 and #781.   There is no test of whether or not the model produces the expected results.   This seems like a  quite a shortcoming given that you instruct a new user of the model to run the example and check whether or not the results are the same as the expected results.

**Comment by jdebacker** [comment]
> This seems like a quite a shortcoming given that you instruct a new user of the model to run the example and check whether or not the results are the same as the expected results.

Yes, we have been careful when deciding on unit tests, especially those running on GH Actions, with processing time in mind, hence the absence of test of a full run of the example (there are already several units tests ensuring the model runs all the way through, even if not the example script).  But you make a good point and a test with a full run and comparing output should be added, even if just on "local" only runs.

**Comment by rickecon** [comment]
This issue is resolved by PR #785. Closing.
